central/main.go

```diff
@@ -354,6 +355,9 @@ func startServices() {
 		administrationEventHandler.Singleton().Start()
 	}
 
+	if err := aws.Singleton().Start(); err != nil {
+		log.Error("Failed to start AWS cloud credentials manager: ", err)
+	}
```

Comment:
Same comment w.r.t. not having the explicit error handling + feature flag as the GCP PR.